### DISCUSSION TITLE
data list  of the strapi admin filter with relation table

### DISCUSSION THREAD

**Discussion by cjfff** [strapi/strapi#6001] [Legacy Topics]
I use strapi with the mysql to build my cms system.

I create some mysql table and create a lot of data. it' s wonderful!

but now i get some trouble when want to change the data.

there are a lot of data...so i want to filter by relation

![image](https://user-images.githubusercontent.com/20502762/80600381-e44c8380-8a5e-11ea-88d5-3bbdd3c19dd5.png)

the feature request just i can use the relation field to search the data list

for example 

my table reaction model is two, can use the filter field to search data list

![image](https://user-images.githubusercontent.com/20502762/80602914-4f4b8980-8a62-11ea-9589-37642659588c.png)

end , the search url is 

`http://localhost:1337/industry-details?_limit=10&_sort=icon:ASC&_start=0&industryExample.name_contains=%E7%A7%80%E5%9F%9F`

![image](https://user-images.githubusercontent.com/20502762/80603815-7c4c6c00-8a63-11ea-977c-f25eab947d34.png)

I want to custom the plugin , but i search the docs found

![image](https://user-images.githubusercontent.com/20502762/80603896-95551d00-8a63-11ea-8bcd-f886b219cd7e.png)

**Comment by cjfff** [comment]
please add the feature or tell me how to custom ~~

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by derrickmehaffy** [comment]
Please see the following entry on the roadmap, this feature is already planned: https://portal.productboard.com/strapi/1-public-roadmap/c/49-display-relational-field-in-list-view
